dockerfiles/ubuntu/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM ubuntu:bionic AS resource
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    curl \
+    ca-certificates \
+    jq \
+  && rm -rf /var/lib/apt/lists/*
+
+ADD assets/ /opt/resource/
+RUN chmod +x /opt/resource/*
+
+FROM resource AS tests
+RUN apt-get update && apt-get install ruby-full ruby-bundler -y
+ADD . /resource
+WORKDIR /resource
+RUN ls -asl /opt/resource
```

Comment:
Hey,

Is there a need for this `ls` here?

Thanks!

```suggestion
```

Reply:
Yeah we also noticed that some of the other commands aren't as efficient as it could be